packages/ui-system/src/components/Button/Button.tsx

```diff
@@ -34,14 +37,17 @@ export const Button = ({
   });
 
   return (
-    <button
-      data-testid={dataTestId}
-      className={`${buttonClasses}`}
-      disabled={disabled || loading}
-      onClick={onClick}
-      type={type}
-    >
-      {loading ? "Loading..." : children}
-    </button>
+    <label htmlFor={id}>
```

Comment:
**issue:** We break semantics by wrapping `button` into `label` in Button which is a part of ui-system